elyra/pipeline/processor_kfp.py

```diff
@@ -37,6 +38,30 @@ class KfpPipelineProcessor(PipelineProcessor):
     def type(self):
         return self._type
 
+    def get_auth_session_cookie(self, url, api_endpoint_username, api_endpoint_password):
```

Comment:
could you make this a private function more towards the bottom `def _get_auth_session_cokkie`... also, should this be called `__get_user_auth_session_cokkie` or `_authorize_user`?